datafusion/sqllogictest/test_files/create_external_table.slt

```diff
@@ -102,8 +102,11 @@ CREATE TEMPORARY TABLE my_temp_table (
     name TEXT NOT NULL
 );
 
+statement ok
+CREATE EXTERNAL TABLE ttemp STORED AS parquet LOCATION '../../parquet-testing/data/alltypes_plain.parquet' WITH ORDER (id);
```

Comment:
the error is now thrown after planning and during execution and has to take a valid table, would it now make sense to layer on where the catalog information is resolved?

Reply:
```sh
CREATE TEMPORARY VIEW my_temp_view AS SELECT id FROM ttemp;
```
I think this test is a duplication of `CREATE TEMPORARY VIEW y AS VALUES (1,2,3)` in ddl.slt. 
Maybe we can remove it.